### PR TITLE
ops-8396 Add S3 bucket policy

### PR DESCRIPTION
**Pull request by daleckystepan**

### CODE REVIEW

**Review by czernitko — DISMISSED** [review]

File: aws/load-balancer.tf [hunk 1]
```diff
@@ -60,6 +60,51 @@ resource "aws_s3_bucket" "rancher_lb_access_logs" {
...
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": "${data.aws_elb_service_account.default.arn}"
+      },
+      "Action": "s3:PutObject",
+      "Resource": "arn:aws:s3:::${var.rancher_lb_access_logs_bucket}/AWSLogs/${data.aws_caller_identity.default.account_id}/*"
```
Comment (gerricchaplin) [inline aws/load-balancer.tf]: AWSLogs may be default but please add explicit prefix AWSLogs on the ALB to make sure it matches.

File: aws/load-balancer.tf [hunk 2]
```diff
@@ -60,6 +60,51 @@ resource "aws_s3_bucket" "rancher_lb_access_logs" {
...
+      "Action": "s3:PutObject",
+      "Resource": "arn:aws:s3:::${var.rancher_lb_access_logs_bucket}/${var.rancher_lb_access_logs_prefix}/AWSLogs/${data.aws_caller_identity.default.account_id}/*"
+    },
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "Service": "delivery.logs.amazonaws.com"
+      },
+      "Action": "s3:PutObject",
+      "Resource": "arn:aws:s3:::${var.rancher_lb_access_logs_bucket}/${var.rancher_lb_access_logs_prefix}/AWSLogs/${data.aws_caller_identity.default.account_id}/*",
```
Comment (gerricchaplin) [inline aws/load-balancer.tf:88]: Best to simplify and change to `arn:aws:s3:::${var.rancher_lb_access_logs_bucket}/${var.rancher_lb_access_logs_prefix}/*`

File: aws/load-balancer.tf [hunk 3]
```diff
@@ -60,6 +60,51 @@ resource "aws_s3_bucket" "rancher_lb_access_logs" {
...
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": "${data.aws_elb_service_account.default.arn}"
+      },
+      "Action": "s3:PutObject",
+      "Resource": "arn:aws:s3:::${var.rancher_lb_access_logs_bucket}/${var.rancher_lb_access_logs_prefix}/AWSLogs/${data.aws_caller_identity.default.account_id}/*"
```
Comment (gerricchaplin) [inline aws/load-balancer.tf:80]: Best to simplify and change to `arn:aws:s3:::${var.rancher_lb_access_logs_bucket}/${var.rancher_lb_access_logs_prefix}/*`

File: aws/load-balancer.tf (hunk 2 above)
Comment (gerricchaplin) [reply aws/load-balancer.tf:88]: I'll accept as is

**Review by gerricchaplin — APPROVED** [review]

**Pull request merged**